docs/exception-handling.md

```diff
@@ -428,19 +428,19 @@ fun main() = runBlocking {
         supervisorScope {
             val child = launch {
                 try {
-                    println("Child is sleeping")
+                    println("The child is sleeping")
```

Comment:
Please also run `./gradlew knit` to update actual `.kt` files

Reply:
It doesn't work, it says `JDK_16 environment variable is not defined.`

Reply:
It's really, really sad that I would have to install JDK 1.6 just to fix some typos :(

Reply:
If the "Allow modifications from maintainers" checkbox is checked, maybe @qwwdfsad can do it?